charts/kyverno/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 type: application
 name: kyverno
-version: v2.5.2
+version: v2.6.0
```

Comment:
Please put the version back.  We will bump the version when a helm chart tag is created.